frontend/src/components/pages/UsersAndGroups/Presentation.jsx

```diff
@@ -201,8 +219,8 @@ export default function Presentation({
           );
         })}
       </Grid>
-      <Grid item xs={6}>
-        <Paper variant="outlined" elevation={2}>
+      <Grid item xs={6} className={classes.root}>
```

Comment:
what does root mean? root generally means the root element. The top-level element.

And why are some grid items named root and some split? 

and what does split mean? And why is is applied inconsistently? 

Remember that good code makes its intentions clear. One way to do that is by using meaningful names.

Please also run the code - the scroll behavior is weird. I think this inconsistency might be the source of the bug.